index.html

```diff
@@ -220,6 +214,7 @@
             </tr>
           </tbody>
         </table>
+        <p class="text-center"><a target="_blank" href="https://github.com/resin-io/etcher#debian-and-ubuntu-based-package-repository-gnulinux-x86x64">Looking for debian images?</a><p>
```

Comment:
Maybe we should also link to the `brew` instructions for Mac? (and `chocolatey` for Windows too, when that feature gets merged?)